src/include/sof/common.h

```diff
@@ -206,15 +206,30 @@
 // IS_ENABLED() above.
 #  if XCHAL_HAVE_HIFI5
 #    define SOF_MAX_XCHAL_HIFI 5
+#    define SOF_FRAME_BYTE_ALIGN	16
+#    define SOF_FRAME_BYTE_ALIGN_6CH	16
 #  elif XCHAL_HAVE_HIFI4
 #    define SOF_MAX_XCHAL_HIFI 4
-#  elif XCHAL_HAVE_HIFI3
+#    define SOF_FRAME_BYTE_ALIGN	8
+#    define SOF_FRAME_BYTE_ALIGN_6CH	16
```

Comment:
I'll move this 6ch specific definition into volume, I can't see generally how 6ch alignement would be a special case. Every word length, channels count has some number of frames that is not matching align with 8 or 16 bytes / 64 or 128 bits.